products/antixlinux.md

```diff
@@ -0,0 +1,60 @@
+---
+permalink: /antix
+alternate_urls:
+-   /antixlinux
+-   /antix-linux
+title: antiX
+category: os
+releasePolicyLink: https://www.antixforum.com/forums/topic/when-is-end-of-support-for-stable-antix-versions-17-19/#post-26424
+iconSlug: NA
+activeSupportColumn: true
+versionCommand: cat /etc/os-release
+releaseColumn: true
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
+purls:
+-   purl: pkg:os/mxlinux
+auto:
+-   distrowatch: antix
+    regex: '^Distribution Release: antiX (?P<major>\d)\.(?P<minor>\d)$'
+releases:
+-   releaseCycle: "21"
+    codename: "Grup Yorum"
+    releaseDate: 2021-10-31
+    latest: "22"
+    latestRelaseDate: 2022-10-19
+    support: 2024-06-30
+    eol: 2026-06-30
+    link: https://antixlinux.com/antix-22-released/
+-   releaseCycle: "19"
+    codename: "Marielle Franco / Hannie Schaft / Manolis Glezos / Grup Yorum"
+    releaseDate: 2019-10-17
+    latest: "19.4"
```

Comment:
according to its website 19.5 exist : https://antixlinux.com/antix-19-5-point-release-update/

Reply:
@usta Thanks and good catch! Please see the updated [antix branch](https://github.com/alexkoster/endoflife.date/tree/antix) for this change.  Thanks for your time and please let me know if any further changes are needed.